src/components/cc-token-oauth-list/cc-token-oauth-list.stories.js

```diff
@@ -0,0 +1,249 @@
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import './cc-token-oauth-list.js';
+
+export default {
+  tags: ['autodocs'],
+  title: '🛠 Profile/<cc-token-oauth-list>',
+  component: 'cc-token-oauth-list',
+};
+
+/**
+ * @typedef {import('./cc-token-oauth-list.js').CcTokenOauthList} CcTokenOauthList
+ * @typedef {import('./cc-token-oauth-list.types.js').OauthTokenState} OauthTokenState
+ */
+
+const conf = {
+  component: 'cc-token-oauth-list',
+};
+
+const ONE_DAY = 24 * 60 * 60 * 1000;
+
+/** @type {Array<OauthTokenState>} */
+const baseTokens = [
+  {
+    type: 'idle',
+    id: '1',
+    consumerName: 'clever-tools',
+    creationDate: new Date(Date.now() - 180 * ONE_DAY),
```

Comment:
nitpick: if you prefer, this can be replaced by using `shiftDateField` date utils: 

```suggestion
    creationDate: shiftDateField(new Date(), 'D', -180),
```